test/test-component.js

```diff
@@ -110,6 +110,15 @@ describe('Simple Component instance', function() {
       expect(el.shadowRoot).to.be.ok;
     });
 
+    it('successfully finds the panel root', function() {
+      el.attachedCallback();
+      childEl = document.createElement('nested-child');
+      childEl.setAttribute('panel-parent', el.panelID);
+      el.el.appendChild(childEl);
+      childEl.attachedCallback();
+      expect(childEl.$panelRoot).to.equal(el);
```

Comment:
well it won't matter since the top level element of the `child` el that has shadow dom is still in the light dom, but we could add a test anyways to prevent some future changes from breaking that